bigquery/api/Snippets/ExtractTable.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright(c) 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+// [START bigquery_extract_table]
+
+using Google.Cloud.BigQuery.V2;
+using System;
+
+public class BigQueryExtractTable
+{
+    public void ExtractTable(
+        string projectId = "your-project-id",
+        string bucketName = "your-bucket-name")
+    {
+        BigQueryClient client = BigQueryClient.Create(projectId);
+        string destinationUri = $"gs://{bucketName}/shakespeare.csv";
```

Comment:
Should we show [a wildcard in the destination URI](https://cloud.google.com/bigquery/docs/exporting-data#exporting_data_into_one_or_more_files) so that the sample works when exporting larger table? That gotcha bites me quite often.

Ditto for the other extract formats.

Reply:
I'll update and make a note to update the other languages. Out of curiosity, what happens when you try to export a larger table without a wildcard?